.eslintrc.json

```diff
@@ -1,12 +1,12 @@
 {
```

Comment:
Spaces have changed to tabs in this file.

Also pointed the eslint at the renamed config package